.eslintrc.js

```diff
@@ -40,4 +41,15 @@ module.exports = {
         'prefer-const': ['error'],
         'semi': ['error', 'always'],
     },
+    overrides: [
+        {
+            files: [
+                '**/*.test.js',
```

Comment:
```suggestion
                'tests/*.test.js',
```
I think we should give tests their own folder. The reason i think we should keep .test.js is because we might need supporting files there too.